nautobot/core/settings.py

```diff
@@ -822,6 +823,11 @@
         help_text="Whether to prefer IPv4 primary addresses over IPv6 primary addresses for devices.",
         field_type=bool,
     ),
+    "RACK_DEFAULT_U_HEIGHT": ConstanceConfigItem(
+        default=RACK_U_HEIGHT_DEFAULT,
```

Comment:
```suggestion
        default=_RACK_U_HEIGHT_DEFAULT,
```

Looks like your updates collided with mine.